web/frontend/src/components/RunConfiguration/Form/RunConfigurationForm.js

```diff
@@ -132,8 +134,8 @@ const RunConfigurationForm = ({
               <>
                 <Title level={5}>General</Title>
                 <FormBlockText style={{ color: colors.grey[3] }}>
-                  Test title would be copied to all new started tests. Use this
-                  field as short test identification to be able quckly
+                  Fields would be copied to all new started tests. Use this
+                  block as short test identification to be able quickly
```

Comment:
```suggestion
                  block as short test identification to be able to quickly
```

Reply:
I think we can remove `to be able` part to make it simplier. Thanks!